README.md

```diff
@@ -78,22 +78,30 @@ $CONFIG = array (
 );
 ```
 ### Usage with [Keycloak](https://www.keycloak.org/)
-1. Create the NextCloud Client in a Keycloak Realm of your choosing.
-	1. Set `Access type` to `confidential`  
-	Note: The `Client Secret` will be in the newly created `Credentials` tab after saving.
-	2. Add a `Valid Redirect uri` of `https://cloud.example.com/*`
-	3. Add a `Mapper` to the Nextcloud OIDC Client 
-		1. Set `Mapper Type` to `user attribute`
-		2. Set `Name`, `User Attribute`, and `Token Claim Name` to `ownCloudQuota`
-		3. Set `Claim JSON Type` as `string`.
-2. Add a User `Attribute` to necessary user accounts in Keycloak.
-	1. Set `Key` as `ownCloudQuota` and `Value` to your preferred limit.
-3. Necessary `config.php` settings (differing from above)
+1. Create a new Client for Nextcloud in a Keycloak Realm of your choosing.
+    1. Set a `Client ID` and save.
+    2. Set `Access type` to `confidential` 
+	3. Add a `Valid Redirect URI` e.g. `https://cloud.example.com/*`.
+	4. Open the  `Fine Grain OpenID Connect Configuration` dropdown and set `ID Token Signature Algorithm` to e.g. `RS512` and save.
+	
+	Note: Only HS256, HS384, HS512, RS256, RS384, RS512 is supported.
+	
+2. Open your created Client and go to `Mappers`.
+    1. Click `create` and set `Mapper Type` to `User Attribute`.
+    2. Set `Name`, `User Attribute`, and `Token Claim Name` to `ownCloudQuota`.
+    3. Set `Claim JSON Type` as `String`.
+   
+3. Add or edit a User and go to `Attributes`. (optional)
+    1. Add an `Attribute` by setting `Key` as `ownCloudQuota` and `Value` to your preferred limit.
```

Comment:
Shouldn't these be merged? In the proposed version, step (2) is not needed, when (3) is not used (and (3) is marked as _optional_).

Reply:
I think the original intention here was to say that the `ownCloudQuota` can be changed for specific users by adding that attribute (it would fall back to the default value in case it is not present). @Excel1 can you revert this change if this makes sense?

Reply:
Oh, my formulation might have been unclear or misleading, sorry. I am not arguing against the change per se, I was just thinking it could be combined into a single item.

Reply:
The steps got sumed up and optimized. Also i included the Group-Feature @pulsejet